docs/tool-description.md

```diff
@@ -1,3 +1 @@
-# Clang Tidy
-
-clang-tidy is a clang-based C++ “linter” tool. Its purpose is to provide an extensible framework for diagnosing and fixing typical programming errors, like style violations, interface misuse, or bugs that can be deduced via static analysis. clang-tidy is modular and provides a convenient interface for writing new checks.
+Clang Tidy is a clang-based C++ “linter” tool. Its purpose is to provide an extensible framework for diagnosing and fixing typical programming errors, like style violations, interface misuse, or bugs that can be deduced via static analysis. clang-tidy is modular and provides a convenient interface for writing new checks.
```

Comment:
They use clang-tidy in their documentation. Maybe we should Change Clang Tidy to Clang-tidy

Reply:
You're right 👍 
Changed the name